ooxml/XWPF/Usermodel/XWPFDocument.cs

```diff
@@ -299,12 +300,20 @@ public CT_Document Document
         /**
          * Sets columns on document base object
          */
-        public void setColumnCount(int num)
+        public int ColumnCount
         {
-            if(ctDocument != null)
+            get
             {
-                ctDocument.body.sectPr.cols.num = num.ToString();
+                return this.columnCountField;
```

Comment:
I think this is an actual error nothing is ever going to come out of this (that I can see) it should be taking and parsing the value back out.

Reply:
Yeah you are right. I forgot to set the columnCountField to value. Is that what you were referring to?